components/benefit_sponsors/config/routes.rb

```diff
@@ -87,6 +87,7 @@
         get :calculate_employee_cost_details, on: :collection
         get :calculate_employee_cost_details, on: :member
         get :reference_product_summary, on: :collection
+        get :estimated_employee_cost_details, on: :collection
```

Comment:
@rahulgudim - This should be at member level. Each benefit application can have more than one benefit package